ingest/vendored/apply-geolocation-rules

```diff
@@ -5,18 +5,6 @@
 any additional transformations on top of the user curations.
 """
 
-"""
-Copied from https://github.com/nextstrain/monkeypox/blob/62fca491c6775573ad036eedf34b271b4952f2c2/ingest/bin/apply-geolocation-rules
-with two changes:
```

Comment:
> The script vendored here contains two code changes

A comment is used to describe the pathogen-specific changes, when those might be better described by git history.

Suggestion: Put these changes in a separate commit. I'd expect something like one commit to add the original contents from `git subtree pull`, then another commit with the pathogen-specific modifications.

Reply:
Oh, I just realized that 7d507f7392f22144900ac3427106709b71ce3270 is a direct copy of https://github.com/nextstrain/ingest/commit/5afbcc788de7a54aafebcc0e2c701d81176a046a and these aren't pathogen-specific changes. You can disregard this suggestion then.

Reply:
> I just realized that https://github.com/nextstrain/hepatitisB/commit/7d507f7392f22144900ac3427106709b71ce3270 is a direct copy of https://github.com/nextstrain/ingest/commit/5afbcc788de7a54aafebcc0e2c701d81176a046a

Yeah, or perhaps the other way around! The ingest commit was created by pushing the subtree from this (hbv) repo. In hindsight, probably not an approach I'd reach for again, but it was a good learning exercise.